framework/src/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -263,24 +265,29 @@ public void standardizeVariable(Tree node, AnnotatedTypeMirror type, Element ele
         }
         switch (ele.getKind()) {
             case PARAMETER:
-                MethodTree methodTree = TreeUtils.enclosingMethod(path);
-                if (methodTree != null) {
+                Tree enclTree =
+                        TreeUtils.enclosingOfKind(
+                                path,
+                                new HashSet<Kind>(
+                                        Arrays.asList(Kind.METHOD, Kind.LAMBDA_EXPRESSION)));
+
+                if (enclTree.getKind() == Kind.METHOD) {
+                    MethodTree methodTree = (MethodTree) enclTree;
                     TypeMirror enclosingType = ElementUtils.enclosingClass(ele).asType();
                     FlowExpressionContext parameterContext =
                             FlowExpressionContext.buildContextForMethodDeclaration(
                                     methodTree, enclosingType, factory.getContext());
                     standardizeDoNotUseLocals(parameterContext, path, type);
-                    break;
+                } else {
+                    // If there is no enclosing method, then the parameter is a parameter to a lambda
```

Comment:
It's not that there is no enclosing method, but that the nearest enclosing thing was a lambda rather than a method.  Am I right about that?
(I realize that you copied the comment from the existing code, but I think it can still be improved.)